L_Arduino.lua

```diff
@@ -218,7 +217,7 @@ function setVariableIfChanged(serviceId, name, value, deviceId)
     if ((value ~= curValue) or (curValue == nil) or (serviceId == "urn:micasaverde-com:serviceId:SceneController1")) then
         luup.variable_set(serviceId, name, value, deviceId)
 		if ((serviceId == "urn:upnp-org:serviceId:TemperatureSetpoint1_Heat") or (serviceId == "urn:upnp-org:serviceId:TemperatureSetpoint1_Cool")) then
-			luup.variable_set("urn:upnp-org:serviceId:TemperatureSetpoint1", name, value, deviceId)
+			luup.variable_set("urn:micasaverde-com:serviceId:LightSensor1", name, value, deviceId)
```

Comment:
Shouldn't this still be urn:upnp-org:serviceId:TemperatureSetpoint1 ?

Reply:
Honestly I don't really know how this code flow works, I'm sorry. I think you are right and, if I understand correctly, it will still use the id 37 for the messages and just overwrite the "type", right?